src/vcpkg/vcpkgcmdarguments.cpp

```diff
@@ -9,6 +9,56 @@
 #include <vcpkg/metrics.h>
 #include <vcpkg/vcpkgcmdarguments.h>
 
+namespace
+{
+    using namespace vcpkg;
+
+    constexpr const std::pair<StringLiteral, StringLiteral> KNOWN_CI_VARIABLES[]{
```

Comment:
```suggestion
    static constexpr std::pair<StringLiteral, StringLiteral> KNOWN_CI_VARIABLES[]{
```

constexpr implies const and static enforces constexpr

Reply:
It's already `static` by virtue of being in an unnamed namespace.